pnpm-lock.yaml

```diff
@@ -6807,9 +6822,12 @@ packages:
   /@types/lodash.isequal@4.5.8:
     resolution: {integrity: sha512-uput6pg4E/tj2LGxCZo9+y27JNyB2OZuuI/T5F+ylVDYuqICLG2/ktjxx0v6GvVntAf8TvEzeQLcV0ffRirXuA==}
     dependencies:
-      '@types/lodash': 4.14.202
```

Comment:
in theory yes but it just aligns all the versions to be same to avoid weird issues in different packages, not very likely but it can happen in some edge cases